lib/private/machines/private/build-mongo-where-clause.js

```diff
@@ -212,6 +212,9 @@ module.exports = function buildMongoWhereClause(whereClause, WLModel, meta) {
 
       case 'like':
         constraint['$regex'] = new RegExp('^' + _.escapeRegExp(modifier).replace(/^%/, '.*').replace(/([^\\])%/g, '$1.*').replace(/\\%/g, '%') + '$');
+        if (meta && meta.makeLikeModifierCaseInsensitiveInMongo && _.isBoolean(meta.makeLikeModifierCaseInsensitiveInMongo)) {
```

Comment:
@frostme this is just a nit, sorry, but would you mind replacing the isBoolean and truthiness check with `meta.makeLikeModifierCaseInsensitiveInMongo === true` instead?

Reply:
@frostme @luislobo I'm ok with the length of `makeLikeModifierCaseInsensitiveInMongo` personally.  It's crazy and long, but this way it's super explicit and hopefully it means no one would get confused  (especially me, since I could see us eventually adding something related to the adapter spec as a built-in thing across all adapters in the future, and I might misunderstand if I'm seeing this out of context)

Anyway, I do agree it's not super practical for the long term to have to type that crazy long thing out in your userland code.  (Really, none of the meta key features are intended to be permanent-- like ideally once we have a way of doing this across the board, you'd be able to chain on a special thing to customize case-sensitivity `.fetch()`)

Reply:
fixed nit with 687b5c3 👍

Reply:
@mikermcneil LGTM